package.json

```diff
@@ -168,6 +168,7 @@
         "babel-preset-stage-0": "6.24.1",
         "babel-register": "6.26.0",
         "babel-watch": "2.0.7",
+        "babili-webpack-plugin": "^0.1.2",
```

Comment:
`babili-webpack-plugin` is deprecated and is now replaced by `babel-minify-webpack-plugin`.

Reply:
Do you use babili in your PR ? If it's the case I will remove it with my next PR.

Reply:
Ok as mentionned IRL, we need it for now. So I will merge the PR and solve the problem in the next PR.

Reply:
Yep, otherwise the dll are not built, maybe we can just modify the render.dev.dll.js file for referencing `babel-minify-webpack-plugin` instead of `babili-webpack-plugin `